README.md

```diff
@@ -1,10 +1,10 @@
-# find-service
+## find-service
 
 Quarkus based backend service for the Find My Relative demo application.
 
-Frontend Service can be found here -`https://github.com/Emergency-Response-Demo/findmyrelative-frontend` 
+Frontend Service can be found [here](`https://github.com/Emergency-Response-Demo/findmyrelative-frontend`). 
```

Comment:
The URL does not get rendered because of ( ` ) surrounding the links.

Reply:
okay will remove it